code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -512,3 +512,53 @@
 	if(index == 1 && target.address == text && target.active)
 		if(target.wearer && target.wearer.stat != DEAD)
 			return target
+
+/*
+	Succour
+	Heals another person, quite powerfully. Only works on NT disciples
+*/
+/datum/ritual/targeted/cruciform/priest/heal_other
+	name = "Succour"
+	phrase = "Venite ad me, omnes qui laboratis, et onerati estis et ego reficiam vos"
+	desc = "Heal a nearby disciple"
+	cooldown = TRUE
+	cooldown_time = 100
+	power = 35
```

Comment:
needs a higher power cost considering it's the best prayer in the game, with the only contender for the same spot being the message ritual

Reply:
I would challenge the idea that succour is as good as you claim, and the cooldown already prevents it from being spammed. Regardless, what power requirement do you propose?

Reply:
i would put a minimum of 50, but it could probably do with being higher when you consider the preacher's higher power capacity and faster regen rate

Reply:
Can do, sorry if I sound passive-aggressive. I'm quite curious as to your reasoning.